engine.js

```diff
@@ -91,7 +91,7 @@ module.exports = function(options) {
             getFromOptionsOrDefaults('jiraPrefix') +
             '-12345):',
           when: options.jiraMode,
-          default: jiraIssue ? jiraIssue.substring(1) : '',
+          default: jiraIssue ? jiraIssue : '',
```

Comment:
This is more idiomatic to JS

```suggestion
          default: jiraIssue || '',
```

Reply:
Ok, i've changed to that. Thanks!